### DISCUSSION TITLE
23.10.20 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#35] [Daily Scrum]
.

**Comment by kdmstj** [comment]
- 오늘은 어떤 일을 했나요?
  - 😇
- 내일은 어떤 일을 하나요?
  - 스포티파이 노래 API 사용 확인
  - PR....

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 😔(스터디 준비..)
- 내일은 어떤 일을 하나요?
  - 비동기 작업 검증
  - PR 작성

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 하얗게 불태운 라이브 코딩...
* 내일은 어떤 일을 하나요?
  * 코드 리팩토링

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 발표 준비
    - 코드리뷰 반영
        - OAuth 정보를 복합키로 사용하도록 변경 (`EmbeddedId`)
        - 메서드 역할 분리
- 내일은 어떤 일을 하나요?
    - 다시 리뷰 후 머지
    - 테스트코드 작성
